File: db/csharp/SalesManagement.Domain/Models/PurchaseOrder.cs
namespace SalesManagement.Domain.Models
{
    /// <summary>
    /// 発注データ
    /// </summary>
    public class PurchaseOrder
    {
        public string PoNo { get; set; } = string.Empty;
        public DateTime PoDate { get; set; }
        public string OrderNo { get; set; } = string.Empty;
        public string SupplierCode { get; set; } = string.Empty;
        public int SupplierBranch { get; set; }
        public string EmployeeCode { get; set; } = string.Empty;
        public DateTime? DueDate { get; set; }
        public string WarehouseCode { get; set; } = string.Empty;
        public int PoAmount { get; set; }
        public int ConsumptionTax { get; set; }
        public string? SlipComment { get; set; }
        public DateTime CreatedAt { get; set; }
        public string CreatedBy { get; set; } = string.Empty;
        public DateTime UpdatedAt { get; set; }
        public string UpdatedBy { get; set; } = string.Empty;
    }
}

File: db/fsharp/SalesManagement.Domain/Models/PurchaseOrder.fs
namespace SalesManagement.Domain.Models

open System

/// <summary>
/// 発注データ
/// </summary>
[<CLIMutable>]
type PurchaseOrder = {
    PoNo: string                   // 発注番号（PK）
    PoDate: DateTime               // 発注日
    OrderNo: string                // 受注番号（FK、追跡用）
    SupplierCode: string           // 仕入先コード（FK）
    SupplierBranch: int            // 仕入先枝番（FK）
    EmployeeCode: string           // 社員コード（FK）
    DueDate: DateTime option       // 指定納期
    WarehouseCode: string          // 倉庫コード（FK）
    PoAmount: int                  // 発注金額合計
    ConsumptionTax: int            // 消費税合計
    SlipComment: string option     // 備考
    CreatedAt: DateTime            // 作成日時
    CreatedBy: string              // 作成者名
    UpdatedAt: DateTime            // 更新日時
    UpdatedBy: string              // 更新者名
}
